examples/example_app/example_app/templates/example_app/config.html

```diff
@@ -24,10 +24,11 @@ <h3>{% block title %}Example App Configuration{% endblock title %}</h3>
                 {% endblock form %}
             </div>
         </div>
-        <div class="row">
-            <div class="col-md-6 col-md-offset-3 text-right">
-                <button type="submit" class="btn btn-primary">Update Configuration</button>
-            </div>
+        <div class="nb-form-sticky-footer">
+            <button type="submit" class="btn btn-primary">
+                <span aria-hidden="true" class="mdi mdi-check me-4"></span><!--
+                -->Update Configuration
```

Comment:
What's the reason for this line-wrapping-with-a-comment pattern? I see you had to update a number of tests to account for it. Is it just for readability of the raw template code, or is there a deeper reason?